tests/wamr-test-suites/spec-test-script/runtest.py

```diff
@@ -148,7 +154,8 @@ def cleanup(self):
                 self.stdout.close()
             self.stdin = None
             self.stdout = None
-            sys.exc_clear()
+            if not IS_PY_3:
+                sys.exc_clear()
```

Comment:
is it worth keeping py2 case?
we already use python3 for all.py.

Reply:
Would you mind sharing your local modification about py2->py3?

Reply:
i was talking about eg. https://github.com/bytecodealliance/wasm-micro-runtime/blob/9d689b3a22d3ccd341393b5f76a071c89879371e/tests/wamr-test-suites/test_wamr.sh#L394
not local modification.

Reply:
For now, `runtest.py` is running under python2.7

Reply:
> For now, `runtest.py` is running under python2.7

yes.
but this PR is making it compatible with py3, isn't it?
my suggestion is to drop py2 support as it would be simpler.
but if you want to keep it for now, it's fine.